selenium-accessibility-examples/pom.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
```

Comment:
@titusfortner does it have to be in it's own module? If it simply uses Selenium then you can just put a file or several files here https://github.com/saucelabs-training/demo-java/tree/master/selenium-examples/src/test/java/com/saucedemo . And add to the main [README](https://github.com/saucelabs-training/demo-java)